src/tls13.c

```diff
@@ -9448,15 +9451,24 @@ int wolfSSL_accept_TLSv13(WOLFSSL* ssl)
 
         case TLS13_ACCEPT_FINISHED_DONE :
 #ifdef HAVE_SESSION_TICKET
-    #ifdef WOLFSSL_TLS13_TICKET_BEFORE_FINISHED
-            if (!ssl->options.verifyPeer) {
-            }
-            else
-    #endif
-            if (!ssl->options.noTicketTls13 && ssl->ctx->ticketEncCb != NULL) {
-                if ((ssl->error = SendTls13NewSessionTicket(ssl)) != 0) {
-                    WOLFSSL_ERROR(ssl->error);
-                    return WOLFSSL_FATAL_ERROR;
+            for (ticketsSent = 0; ticketsSent < ssl->options.maxTicketTls13;
+                    ticketsSent++) {
+        #ifdef WOLFSSL_TLS13_TICKET_BEFORE_FINISHED
+                if (!ssl->options.verifyPeer) {
```

Comment:
We only write out one ticket if we don't verify peer and WOLFSSL_TLS13_TICKET_BEFORE_FINISHED is defined.
Is this intended?
I would have expected at least the previous one to be counted and more sent after FINISHED sent.